src/aes.h

```diff
@@ -24,13 +24,20 @@
 
 namespace toit {
 
-class AesCbcContext : public SimpleResource {
+class AesEcbContext : public SimpleResource {
  public:
-  TAG(AesCbcContext);
-  AesCbcContext(SimpleResourceGroup* group, const uint8* key, const uint8* iv, bool encrypt);
-  ~AesCbcContext();
+  TAG(AesEcbContext);
+  AesEcbContext(SimpleResourceGroup* group, const Blob* key, bool encrypt);
+  virtual ~AesEcbContext();
 
   mbedtls_aes_context context_;
+};
+
+class AesCbcContext : public AesEcbContext {
```

Comment:
Does this inheritance make sense conceptually? Should they really have a shared super class? At least add a comment to explain the choice.

Reply:
I changed the name to AesContext and made a comment about ECB mode.